lib/susedistribution.pm

```diff
@@ -333,7 +331,7 @@ sub ensure_installed {
             next;
         }
     }
-    my $ret = wait_serial('pkcon-status-\d+');
+    my $ret = wait_serial('pkcon-status-\d+', timeout => $args{timeout});
```

Comment:
Here, we are waiting for the end of the installation and 90 seconds is not always enough.

Reply:
I don't think so. There is a while loop on before that waits for pkcon finished. After that we only need to wait for the process to exit after it was already done with installation

Reply:
No, the while loop can exit _before_ the installation is done.

Reply:
It seems like the `pkcon-finished` needle simply needs updating, it doesn't match anymore.

Reply:
This seems more likely, yes.

So maybe it makes sense to just introduce a different fail message here with a hint to the needle

Reply:
There is no screenshot usable to update the needle easily from the webui to check this assumption. Maybe we should have an `assert_screen 'pkcon-finished'` when leaving the while loop?

Reply:
> There is no screenshot usable to update the needle easily from the webui to check this assumption.

There should be - it just matches `pkcon-status` in xterm.

It could just use `assert_screen` in line 319